README.md

```diff
@@ -277,3 +278,72 @@ See the [metrics documentation](metrics).
 
 ## Kata Admission controller webhook
 See the [webhook documentation](kata-webhook).
+
+## Using Vagrant to test your code changes
+
+It is strongly recommended that you test your changes locally before opening
+a pull request as this can save people's time and CI resources. Because
+testing Kata Containers involve complex build and setup instructions, scripts
+on the `.ci` directory are created to ease and provide a reproducible process; but they
+are meant to run on CI environments that can be discarded after use. Therefore,
+developers have noticed dangerous side effects from running those scripts on a workstation
+or development environment.
+
+That said, we provide in this repository a `Vagrantfile` which allows developers to use
+the [vagrant](https://www.vagrantup.com) tool to create a VM with the setup as close as
+as possible to the environments where CI jobs will run the tests. Thus, allowing to
+reproduce a CI job locally.
+
+Currently it is only able to create a *Fedora 32* or *Ubuntu 20.04* VM. And your workstation
```

Comment:
Since F32 is dead, can we move to F33 yet?

Reply:
I'm using F32 here because it is the version used on CI (I want to keep the versions in sync). But I agree we should bump to 33 or 34, https://github.com/kata-containers/ci/issues/389